### PR TITLE
Feature | adding bucket name output.

### PR DESCRIPTION
**Pull request by mvago-flx**
## what
* Add an output for  bucket name on the layer

## why
* Need to use bucket name on the layer instead bucket_arn or both

## references
* GitHub issue [`#13` ](https://github.com/binbashar/terraform-aws-vpc-flowlogs/issues/13)

### CODE REVIEW

**Review by lgallard — APPROVED** [review]
LGTM!

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (exequielrafaela) [inline yarn.lock:4]: @mvago-flx Tincho let's please remove this file. Thanks :) 

CC: @lgallard

**Review by exequielrafaela — CHANGES_REQUESTED** [review]

**Review by lgallard — APPROVED** [review]
LGTM!

File: yarn.lock (hunk 1 above)
Comment (mvago-flx) [reply yarn.lock]: ready

**Pull request merged**